src/modules/modifiers/curve.rs

```diff
@@ -55,14 +56,16 @@ impl<'a, T, U> Curve<'a, T, U>
             let insertion_point = self.control_points
                 .iter()
                 .position(|x| x.input >= input_value)
-                .unwrap_or(self.control_points.len());
+                .unwrap_or_else(|| self.control_points.len());
```

Comment:
`unwrap_or` always executes the `self.control_points.len()` and discards it if it isn't needed, the closure defers execution. In this situation is almost certainly doesn't matter because it's just reading a field but the lint is generic.